lib/blacklight/solr/search_builder_behavior.rb

```diff
@@ -190,6 +194,30 @@ def add_group_config_to_solr solr_parameters
       end
     end
 
+    def add_facet_paging_to_solr(solr_params)
```

Comment:
Assignment Branch Condition size for add_facet_paging_to_solr is too high. [43.34/15]